packages/venia-ui/lib/RootComponents/Category/category.js

```diff
@@ -98,7 +95,7 @@ const Category = props => {
                 filters: newFilters,
                 onServer: false,
                 pageSize: Number(pageSize),
-                sort: { [String(sortAttribute)]: String(sortDirection) }
+                sort: { [currentSort.sortAttribute]: currentSort.sortDirection }
```

Comment:
I wonder if we couldn't modify the `pageControl` to just take `currentSort` and do this so that callers don't need to know about it.

Reply:
@supernova-at i can try to do it but this will expand the scope much

Reply:
Yep, definitely